bundle/metadata/annotations.yaml

```diff
@@ -0,0 +1,15 @@
+annotations:
+  # Core bundle annotations.
+  operators.operatorframework.io.bundle.mediatype.v1: registry+v1
+  operators.operatorframework.io.bundle.manifests.v1: manifests/
+  operators.operatorframework.io.bundle.metadata.v1: metadata/
+  operators.operatorframework.io.bundle.package.v1: nic-configuration-operator
+  operators.operatorframework.io.bundle.channels.v1: v1.1,stable
+  operators.operatorframework.io.bundle.channel.default.v1: v1.1
+  operators.operatorframework.io.metrics.builder: operator-sdk-v1.37.0
+  operators.operatorframework.io.metrics.mediatype.v1: metrics+v1
+  operators.operatorframework.io.metrics.project_layout: go.kubebuilder.io/v4
+
+  # Annotations for testing.
+  operators.operatorframework.io.test.mediatype.v1: scorecard+v1
+  operators.operatorframework.io.test.config.v1: tests/scorecard/
```

Comment:
Let's add supported OCP versions. E.g.:   `com.redhat.openshift.versions: v4.14-v4.16`

Reply:
Added this annotation in the Makefile